earth_enterprise/BUILD_RPMS.md

```diff
@@ -27,6 +27,12 @@ java-1.7.0-openjdk-devel```.  There are no other special prerequisites, as the
 SCons build target pulls down osPackage and dependencies as needed before
 creating RPMs.
 
+Ensure that all required Debian or Yum repositories are enabled on your 
+platform.  For EL6 and EL7 systems, the EPEL repository must be installed 
+and enabled. 
+RHEL satellite based systems also require the *optional* repository ( i.e., rhui-DISTRO-rhel-server-releases-optional) repository to provide some libraries 
```

Comment:
Minor: Can you include a complete command for enabling a repository with `subscription-manager`?  I don't think I've seen a `rhui-DISTRO` before.

Reply:
Yes - we should do this, but it's a little tricky because there are many variations.  The AWS GovCloud environment I tested on had `rhui-REGION-rhel-server-releases-optional`, RHEL6 has "rhel-6-server-optional-rpms", but that can vary depending on whether it's RHSN, RHN satellite, or "Classic".   I don't believe this should hold up this PR